src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/health/PingHealthCheck.kt

```diff
@@ -0,0 +1,85 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.health
+
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Tags
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.actuate.health.Health
+import org.springframework.boot.actuate.health.HealthIndicator
+import org.springframework.boot.actuate.health.Status
+import org.springframework.http.HttpStatusCode
+import org.springframework.http.ResponseEntity
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Mono
+import java.time.Duration
+import java.util.concurrent.atomic.AtomicInteger
+
+abstract class PingHealthCheck(
```

Comment:
What's this for? Doesn't seem to be referenced anywhere else in the PR

Reply:
Done